plugin-cgroupv2/src/k8s/token.rs

```diff
@@ -38,13 +39,12 @@ impl Token {
 
     /// Check if the token' lifetime has reached its expiration or not.
     async fn is_valid(&self) -> bool {
-        let now_secs = match SystemTime::now().duration_since(UNIX_EPOCH) {
-            Ok(ts) => ts.as_secs(),
-            Err(_) => return false,
-        };
+        let now_secs: u64 = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .expect("ERROR : Failed to get the current time since UNIX_EPOCH")
+            .as_secs();
 
-        // Lock the data for the entire check.
-        let data_locked = self.data.read().await;
+        let data_locked: tokio::sync::RwLockReadGuard<'_, TokenData> = self.data.read().await;
```

Comment:
?

Reply:
pro-tip: you can configure VSCode to display the types of the variables on demand.
![image](https://github.com/user-attachments/assets/b0c1cb53-3a78-4a24-ac3a-7b37730ca609)